components/lcd/esp_lcd_lt8912b/README.md

```diff
@@ -0,0 +1,24 @@
+# ESP LCD LT8912B
+
+[![Component Registry](https://components.espressif.com/components/espressif/esp_lcd_lt8912b/badge.svg)](https://components.espressif.com/components/espressif/esp_lcd_lt8912b)
+
+Implementation of the LT8912B LCD controller with esp_lcd component.
```

Comment:
Not a blocker for merging: this description could be extended to say that this is not really an LCD controller but  a MIPI-HDMI bridge.